learners/reference.md

```diff
@@ -1,7 +1,1292 @@
+    estándar del retraso ([Park et al., en
+    progreso](https://github.com/parksw3/epidist-paper)). Diferentes
+    enfoques de muestreo pueden generar sesgos debido a la censura a la
+    izquierda y a la derecha en la estimación del intervalo serial, lo
+    que puede propagar el sesgo a la estimación del período de
+    incubación y el tiempo de generación ([Chen et al.,
+    2022](https://www.nature.com/articles/s41467-022-35496-8/figures/2)).
+
+# Riesgos Relativos
+
+[Risk Ratio (RR)]{#rr}
+
+:   La medida Risk Ratio (RR) también llamada Razón de riesgo o riesgo
+    relativo, compara el riesgo de desarrollar un evento de salud
+    (enfermedad, muerte) entre dos grupos Lo hace dividiendo el riesgo
+    (proporción de incidencia, tasa de ataque) en el grupo 1 por el
+    riesgo (proporción de incidencia, tasa de ataque) en el grupo 2. Los
+    dos grupos generalmente se diferencian por factores demográficos
+    como el sexo o por exposición a un factor de riesgo sospechoso, por
+    ejemplo, consumo de alimento o contacto con casos sospechosos. 
+
+La fórmula para la razón de riesgo (RR) es:
+
+Riesgo de enfermedad (proporción de incidencia, tasa de ataque) en el
+grupo de interés primarioRiesgo de enfermedad (proporción de incidencia,
+tasa de ataque) en el grupo de comparación
+
+De ahí se puede tener que: 
+
+-   Si RR es igual a 1 indica un riesgo idéntico entre los dos grupos. 
+
+-   Si RR es mayor a 1 indica un mayor riesgo para el grupo del
+    numerador, generalmente el grupo expuesto.
+
+-   Si RR es menor a 1 indica una disminución del riesgo para el grupo
+    expuesto, lo que indica que tal vez la exposición en realidad
+    protege contra la aparición de enfermedades.
+
+**Ejemplo**
+
+-   En un brote de un virus respiratorio entre estudiantes de una
+    escuela, 28 de 157 estudiantes de primaria desarrollaron
+    tuberculosis, en comparación con 4 de 137 estudiantes de secundaria.
+    Definiendo la exposición cómo ser estudiante de la sede primaria, y
+    la no exposición ser estudiante de la sede secundaria, se tiene
+    que: 
+
++--------------------------------+----------------+------------+------------+
+|                                | **Enf e rmos** | **S anos** | **T otal** |
++--------------------------------+----------------+------------+------------+
+| **Estu d iantes pri m aria**   | 30             | 120        | 150        |
++--------------------------------+----------------+------------+------------+
+| **Estu d iantes secun d aria** | 5              | 135        | 140        |
++--------------------------------+----------------+------------+------------+
+| -   \*To t                     | 35             | 265        | 290        |
+|                                |                |            |            |
+| al\*\*                         |                |            |            |
++--------------------------------+----------------+------------+------------+
```

Comment:
with https://www.tablesgenerator.com/markdown_tables# we build a fix tables for markdown

```suggestion
|  | Enfermos | Sanos | Total |
|---|---|---|---|
| **Estudiantes primaria** | 30 | 120 | 150 |
| **Estudiantes secondaria** | 5 | 135 | 140 |
| Total | 35 | 265 | 290 |
```